### PR TITLE
removes the removal of patches and sprays not working on thickmaterial stuff because it apparently broke half the game

### PR DESCRIPTION
**Pull request by ghost**
you could easily have simply made it so that hardsuits and SPECIFICALLY hardsuits had a separate flag that disallowed for patch use and then had applications for patches check for that instead so that it didn't work on hardsuits but worked on everything else that required it

instead, as a result of this: golems, skeletons, and ninjas et cetera cannot heal, and ninjas can't heal at all without removing their entire suit. you have to use pills instead, and that's not intended behavior.

it's not worth breaking 2 species and 1 antag for a mostly irrelevant balance decision that was ALREADY RESOLVED by styptic powder and silver sulfadiazine gaining an overdose limit regardless to prevent abuse an consistent spam

and i should mention, this actually DOESN'T even fix the intended issue to start with, because you can bypass the entire problem by literally just splashing a beaker of styptic powder or silver sulfadizine or synthflesh ON YOURSELF in the hardsuit because while the PATCH won't apply and the SPRAY won't apply the beakersplash will

in short this was a lazy PR that broke several features that stayed in to deter 'powergame' when in reality most of the best antibrute chem mixes can be taken in pill form regardless and it could be bypassed with beaker splashing and instead simply broke an antagonist and two species (one of which of those species is used by two other separate antags, pirates and liches)

i understand you guys don't like powergame but if you're going to merge stuff like this at the very least don't half-ass it and instead do it properly, because a change that breaks half the game isn't worth merging

### CODE REVIEW

**Comment by Reed0506** [comment]
i can confirm that the original PR was bad or something, and was exceptionally inconvenient except when in a hardsuit, where it didnt matter to me at all.

**Comment by Moja10** [comment]
What is the original PR you are referencing?

**Comment by JamieD1** [comment]
I shall be reworking this instead of reverting it with our comments in DM

**Review by JamieD1 — APPROVED** [review]

**Comment by Hopekz** [comment]
Can we get a counter going for the amount of times people merge something thinking they'll fix it later and it never gets fixed?
